lib/xboxrt/vcruntime/excpt.cpp

```diff
@@ -0,0 +1,184 @@
+#include <excpt.h>
+#include <eh.h>
+#include <xboxkrnl/xboxkrnl.h>
+
+typedef struct _EXCEPTION_POINTERS
+{
+    PEXCEPTION_RECORD ExceptionRecord;
+    PCONTEXT ContextRecord;
+} EXCEPTION_POINTERS, *PEXCEPTION_POINTERS;
+
+struct ScopeTableEntry
+{
+    DWORD EnclosingLevel;
+    void *FilterFunction;
+    void *HandlerFunction;
+};
+
+struct EXCEPTION_REGISTRATION_SEH3 : EXCEPTION_REGISTRATION
+{
+    ScopeTableEntry *ScopeTable;
+    DWORD TryLevel;
+    DWORD _ebp;
+};
+
+#define DISPOSITION_DISMISS 0
+#define DISPOSITION_CONTINUE_SEARCH 1
+#define DISPOSITION_NESTED_EXCEPTION 2
+#define DISPOSITION_COLLIDED_UNWIND 3
+#define TRYLEVEL_NONE -1
+
+extern "C" int _nested_unwind_handler (_EXCEPTION_RECORD *pExceptionRecord, EXCEPTION_REGISTRATION_SEH3 *pRegistrationFrame, _CONTEXT *pContextRecord, EXCEPTION_REGISTRATION **pDispatcherContext)
```

Comment:
nit: The other `.cpp` files in the vcruntime directory seems to use a "more global" `extern "C"` rather than per-function.